proto/greptime/v1/index/inverted_index.proto

```diff
@@ -63,6 +63,9 @@ message InvertedIndexMeta {
 
     // Statistical information about the tag's inverted index.
     InvertedIndexStats stats = 8;
+
+    // The type of bitmap used for indexing.
+    BitmapType bitmap_type = 9;
```

Comment:
You can resolve this after https://github.com/GreptimeTeam/greptimedb/pull/5603 is ready to merge.